### PR TITLE
Fix duplicated screenshots in the Fine grain admin permissions sectio…

### LINKED ISSUE #31083: Docs: server_admin/topics/admin-console-permissions/fine-grain.adoc
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

docs

### Describe the bug

In the fine grained access permissions documentation there is a duplicate image

File: server_admin/topics/admin-console-permissions/fine-grain.adoc

The images in question are:
 - https://github.com/keycloak/keycloak/blob/4edbd1549eda4a270603923ebcb363f163110cde/docs/documentation/server_admin/images/fine-grain-client-assign-user-policy.png
 - https://github.com/keycloak/keycloak/blob/4edbd1549eda4a270603923ebcb363f163110cde/docs/documentation/server_admin/images/fine-grain-client-user-policy.png

### Version

25.0.1

### Regression

- [ ] The issue is a regression

### Expected behavior

Image shows the required steps to add the policy to the user

### Actual behavior

Image shows previous step and there isn't a complete description of where the user must go or what the user must do

### How to Reproduce?

Visit https://www.keycloak.org/docs/latest/server_admin/#managing-one-specific-client and scroll down to the `Assign user policy` screen

### Anything else?

Without this image it's really hard to get a complete picture of the process.

### PR DESCRIPTION
**Pull request by sguilhen**
…n of the Server Admin Guide

Closes #31083

### CODE REVIEW

File: docs/documentation/server_admin/topics/admin-console-permissions/fine-grain.adoc [hunk 1]
```diff
@@ -68,9 +68,9 @@ image:images/fine-grain-client-manage-permissions.png[Fine grain client manage p
 
 When first initialized the `manage` permission does not have any policies associated with it.
 You will need to create one by going to the policy tab.  To get there fast, click on
-the `Authorization` link shown in the above image. Then click on the policies tab.
+the `Client details` link shown in the above image. Then click on the policies tab.
 
-There's a pull down menu on this page called `Create policy`.  There's a multitude of policies
+There's a button on this page called `Create client policy`.  There's a multitude of policies
 you can define.  You can define a policy that is associated with a role or a group or even define
 rules in JavaScript.  For this simple example, we're going to create a `User Policy`.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/admin-console-permissions/fine-grain.adoc:75]:
```suggestion
On this page, look for the `Create client policy` button, which you can use to define many policies.  You can define a policy that is associated with a role or a group or even define
rules in JavaScript.  For this simple example, we are going to create a `User Policy`.
```

File: docs/documentation/server_admin/topics/admin-console-permissions/fine-grain.adoc [hunk 2]
```diff
@@ -83,15 +83,13 @@ This policy will match a hard-coded user in the user database.  In this case, it
 .Assign user policy
 image:images/fine-grain-client-assign-user-policy.png[Fine grain client assign user policy]
 
-The `sales-admin` user can now has permission to manage the `sales-application` client.
+The `sales-admin` user now has permission to manage the `sales-application` client.
 
-There's one more thing we have to do.  Go to the `Role Mappings` tab and assign the `query-clients`
-role to the `sales-admin`.
+There's one more thing we have to do.  Go to `Users`, select the `sales-admin` user, then go to the `Role Mappings` tab and assign the `query-clients` role to the user.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/admin-console-permissions/fine-grain.adoc:88]:
```suggestion
There is one more thing we have to do.  Go to `Users`, select the `sales-admin` user, then go to the `Role Mappings` tab and assign the `query-clients` role to the user.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks for fixing this lost issue, @sguilhen

**Review by andymunro — APPROVED** [review]
LGTM, @sguilhen

**Comment by sguilhen** [comment]
Please squash the commits when merging.

**Review by ahus1 — APPROVED** [review]
Approving as of @andymunro's review

**Pull request merged**